ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -477,6 +504,71 @@ private void OnViewChanging(object sender, ScrollViewerViewChangingEventArgs arg
                         }));
         }
 
+        private void OnLoaded(object sender, RoutedEventArgs e)
+        {
+            var scrollViewer = (ScrollViewer)sender;
+            _scrollViewerData[scrollViewer].IsControlLoaded = true;
+
+            UpdateDisableKeyboardBasedScrolling(scrollViewer, _scrollViewerData[scrollViewer].DisableArrowNavigation);
+        }
+
+        private void UpdateDisableKeyboardBasedScrolling(ScrollViewer view, bool disable)
+        {
+            ScrollContentPresenter contentPresenter = _scrollViewerData[view].ContentPresenter;
+            if (disable)
+            {
+                if (contentPresenter == null)
+                {
+                    contentPresenter = FindChild<ScrollContentPresenter>(view);
+                    if (contentPresenter == null)
+                    {
+                        throw new InvalidOperationException("ScrollViewer doesn't seem to contain a ScrollContentPresenter");
+                    }
+                    _scrollViewerData[view].ContentPresenter = contentPresenter;
+                }
+
+                contentPresenter.KeyDown += OnPresenterKeyDown;
+            }
+            else
+            {
+                if (contentPresenter != null)
+                {
+                    contentPresenter.KeyDown -= OnPresenterKeyDown;
+                }
+            }
+
+            _scrollViewerData[view].DisableArrowNavigation = disable;
+        }
+
+        internal static T FindChild<T>(DependencyObject startNode)
```

Comment:
>internal [](start = 8, length = 8)

reason for internal? (maybe I need to read more code...)

Reply:
Perhaps a mistake.

Reply:
Perhaps a mistake.

---
In reply to: [155706133](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r155706133) [](ancestors = 155706133)